pkg/kubectl/cmd/util/env/env_parse_test.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package env
+
+import (
+	"fmt"
+)
+
+func ExampleEnv_defaultValue() {
+	fmt.Println(Env("THISENVVARDOESNOTEXIST", "default"))
+	// Output: default
+}
+
+func ExampleEnv_variableExists() {
+	fmt.Println(Env("LANG", "default"))
+	// Output: en_US.UTF-8
```

Comment:
I don't expect this to work for everyone, should I?

Reply:
Ah, yeah, good point. I'll just set a random variable and unset it afterwards. Thanks.

Reply:
sgtm